src/modules/kraken.js

```diff
@@ -0,0 +1,13 @@
+export default class Kraken {
+  scores = [];
+
+  gameID = 'rXsiRnCQjdB5JNjPRRQN';
```

Comment:
[OPTIONAL]

I think it will be better if you dont expose your API key (this is a secret key) on Github to prevent any malicious activity on your codebase 👍🏽

It's always best to store your secret keys to environment variables so that they won't be exposed on GitHub. This way, your app will be more secure 😄

In your free time you can checkout how to add environment variable using webpack

Reply:
Thank you for the review and approval I will kindly store the API key in the environment variables to prevent malicious  activity in my code base 👍🏽 @vickymarz